src/event/kqueue.rs

```diff
@@ -155,6 +155,39 @@ impl Event {
     }
 }
 
+impl Default for Event {
+    fn default() -> Self {
+        Event {
+            inner: kevent_t {
+                ..unsafe { zeroed() }
+            },
+        }
+    }
+}
```

Comment:
As `Event` implements `Clone` I think this is technically not required, as we can just do the zero'ing in the `Events` type. This is in fact probably better such that users can't create a zero'd (and thus basically invalid) `Event` type themselves.